Specs/Number/English/NumberModel.json

```diff
@@ -2700,5 +2700,37 @@
         "End": 3
       }
     ]
+  },
+  {
+    "Input": "12,34,567.89",
```

Comment:
Can you add a proper sentence, so there's a little more context for the spec case?
Also, please add a "Comment" in the spec to document that this is because of the Indian numbering system and the format may not need to be supported in other languages.

Reply:
done